src/main/java/com/byznass/tiolktrack/jaxrs/resource/TestResource.java

```diff
@@ -0,0 +1,43 @@
+package com.byznass.tiolktrack.jaxrs.resource;
+
+import com.byznass.tiolktrack.postgres.ConnectionProvider;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+@Path("/test")
+public class TestResource {
+
+	private final ConnectionProvider connectionProvider;
+
+	@Inject
+	public TestResource(ConnectionProvider connectionProvider) {
+
+		this.connectionProvider = connectionProvider;
+	}
+
+	@GET
+	public String getAllLine() {
+
+		try (Connection connection = connectionProvider.getConnection()) {
+			Statement statement = connection.createStatement();
+			ResultSet resultSet = statement.executeQuery("SELECT * FROM testTable");
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use try-with-resources or close this "ResultSet" in a "finally" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3AS2095)